packages/hash/api/src/model/knowledge/org.model.ts

```diff
@@ -82,14 +81,30 @@ export default class extends EntityModel {
   }
 
   /**
-   * Get the system Organization entity type.
+   * Get a workspace organization entity by its entity id.
+   *
+   * @param params.entityId - the entity id of the organization
    */
-  static async getOrgEntityType(graphApi: GraphApi) {
-    const versionedUri = WORKSPACE_TYPES.entityType.org.schema.$id;
-
-    return await EntityTypeModel.get(graphApi, {
-      versionedUri,
+  static async getOrgById(
+    graphApi: GraphApi,
+    params: { entityId: string },
+  ): Promise<OrgModel | null> {
+    const entity = await EntityModel.getLatest(graphApi, {
+      // assumption: `accountId` of organizations is always the workspace account id
+      accountId: workspaceAccountId,
+      entityId: params.entityId,
     });
+
+    if (
+      entity.entityTypeModel.schema.$id !==
+      WORKSPACE_TYPES.entityType.org.schema.$id
+    ) {
+      throw new Error(
+        `Entity with id ${params.entityId} is not a workspace org`,
+      );
+    }
+
+    return entity ? new OrgModel(entity) : null;
```

Comment:
How can this happen? You throw an error above, I think `entity.entityTypeModel.schema.$id !== WORKSPACE_TYPES.entityType.org.schema.$id` would always be false if `entity` is nullish

Reply:
This could've been a bug, so good spot, but the reason I didn't get a type error when calling `entity.entityTypeModel.schema.$id` is before the null check is because `EntityModel.getLatest` doesn't ever return null. I've therefore modified these methods to also not return null in https://github.com/hashintel/hash/pull/1068/commits/0d6dbac7a59abb5bac60a5dbfdfd490c9bd6d0d3